src/dataframe/dataframe.jl

```diff
@@ -258,18 +268,22 @@ ncol(df::DataFrame) = length(index(df))
     @inbounds cols[col_ind]
 end
 
+# TODO: this will be deprecated and replaced by getcol function
```

Comment:
Can you leave this out for now? Better tackle one issue at a time. Same below.

Reply:
OK